File: ccp5/src/main/java/com/ccp5/dto/IngrBoard.java
package com.ccp5.dto;




import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "ingredients_board")
@Getter
@Setter
public class IngrBoard {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="inum")
    private int id;
    private String title;
    private String category;
    private String name;
    private int unit;
}

File: cc/ccp/app/src/main/java/com/example/ccp/model/IngrBoard.kt
package com.example.ccp.model



data class IngrBoard(

    var id: Int = 0,
    var title: String? = null,
    var category: String? = null,
    var name: String? = null,
    var unit: Int = 0,
    var imageUrl: String? = null
) {
    // 생성자
    constructor(title: String, category: String, name: String, unit: Int) : this() {
        this.title = title
        this.category = category
        this.name = name
        this.unit = unit
    }
}
